docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -171,7 +171,10 @@ Prior to the 8.6 release, Play can be accessed by installing the 8.6.0-alpha [He
 
 ### Features
 
-[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- [Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- Currently, Play supports displaying up to 100 flow node instances in the instance history panel, 100 variables in the variables panel, and 100 process instances on the process definition page. To access all related data, you can use Operate.
+- While you can still interact with your process instance in Play (for example, completing jobs or publishing messages), you may be unable to resolve incidents if they occur beyond the 100th flow node instance, as Play does not track them. In this case, incident resolution can be managed in Operate.
+- Play doesn't support call activities with [FEEL expressions](/components/modeler/feel/what-is-feel.md).
```

Comment:
Maybe we can generalize this a bit more, but I don't know what the proper wording would be. Or maybe we just list all the items: 
Play does not support Job Types defined through feel expressions, Message correlation keys defined by feel expressions, called process definitions defined by Feel expressions. Maybe I forgot something 😅